polaris-core/src/main/java/org/apache/polaris/core/config/RelationalJdbcConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.core.config;
+
+import io.smallrye.config.ConfigMapping;
+import java.util.Map;
+import java.util.Optional;
+
+@ConfigMapping(prefix = "polaris.relation.jdbc.datasource")
```

Comment:
I would rather remove the annotation here and create a sub-interface in a Quarkus module with the annotation.

Reply:
Hmm in fact, this interface is only used in Quarkus modules, so I'd suggest moving it to `quarkus-commons`.

Reply:
Yes, good point about this config being internal to Quarkus modules.